var/spack/repos/builtin/packages/gcc/package.py

```diff
@@ -115,7 +115,7 @@ class Gcc(AutotoolsPackage, GNUMirrorPackage):
     depends_on('zstd', when='@10:')
     depends_on('iconv', when='platform=darwin')
     depends_on('gnat', when='languages=ada')
-    depends_on('binutils~libiberty', when='+binutils')
+    depends_on('binutils~libiberty', when='+binutils', type=('build', 'run'))
```

Comment:
```suggestion
    depends_on('binutils~libiberty', when='+binutils', type=('build', 'link', 'run'))
```
I would include `link` just to be sure.